src/ConfigEditor/Connection/README.md

```diff
@@ -0,0 +1,64 @@
+# Connection
+
+Contains components to be used for datasource connection configuration.
+
+## ConnectionSettings
+
+### Overview
+
+`ConnectionSettings` component is used to simplify migrating from legacy `DataSourceHttpSettings` component.
+Currently renders as a config section with only on field for the URL.
+
```

Comment:
Maybe make it clear that if someone is not migrating over (has other fields), that they shouldn't use this component?

Reply:
Done in a3fcb9bd1b9aaa410b4d4f830e16f53db89da98a.